IntegrationTests/src/bkr/inttest/labcontroller/test_watchdog.py

```diff
@@ -317,6 +317,77 @@ def test_handles_non_ascii_characters_in_openstack_console(self, test_get_consol
         self.assertEquals(open(self.cached_console_log).read().decode('utf8'), log_contents)
 
 
+# Class WatchdogGuestConsoleLogTest tests console log handling for Guest Recipes
+class WatchdogGuestConsoleLogTest(TestHelper):
+    @classmethod
+    def setUpClass(cls):
+        makedirs_ignore(get_conf().get('CONSOLE_LOGS'), 0755)
```

Comment:
`0755` -> `0o755`

Reply:
Changed it here as well as the code I copied it from.